openapi/components/schemas/GatewayAccount.yaml

```
@@ -256,6 +256,10 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  dccForceRounding:
+    type: boolean
+    default: false
+    description: Specifies that DCC quote amounts must be rounded half up to the nearest non-decimal amount.
```

```suggestion
    description: Specifies that DCC quote amounts must be rounded half up to the nearest whole number.
```
?

`nearest non-decimal amount` seems incorrect

---

Good point